samples/src/main/java/StreamingIngest.java

```diff
@@ -0,0 +1,129 @@
+import com.microsoft.azure.kusto.data.ConnectionStringBuilder;
+import com.microsoft.azure.kusto.ingest.IngestionMapping;
+import com.microsoft.azure.kusto.ingest.IngestionProperties;
+import com.microsoft.azure.kusto.ingest.StreamingIngestClient;
+import com.microsoft.azure.kusto.ingest.result.OperationStatus;
+import com.microsoft.azure.kusto.ingest.source.BlobSourceInfo;
+import com.microsoft.azure.kusto.ingest.source.FileSourceInfo;
+import com.microsoft.azure.kusto.ingest.source.StreamSourceInfo;
+import com.microsoft.azure.kusto.ingest.IngestClientFactory;
+
+import java.io.*;
+import java.nio.charset.Charset;
+
+public class StreamingIngest {
+
+    static private String database;
+    static private String table;
+    static private String mapping;
+
+    private static ConnectionStringBuilder csb;
+    private static StreamingIngestClient streamingIngestClient;
+
+    public static void main(String[] args) {
+        try {
+            csb = ConnectionStringBuilder.createWithAadApplicationCredentials(
+                    System.getProperty("clusterPath"), // "https://<cluster>.kusto.windows.net"
+                    System.getProperty("app-id"),
+                    System.getProperty("appKey"),
+                    System.getProperty("tenant"));
+            streamingIngestClient = IngestClientFactory.createStreamingIngestClient(csb);
+
+            database = System.getProperty("dbName");
+            table = System.getProperty("tableName");
+            mapping = System.getProperty("dataMappingName");
+
+            IngestFromStream();
+            IngestFromFile();
+
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    static void IngestFromStream() throws Exception {
```

Comment:
I think there're too much samples here which are very similar. I'd put just a couple of examples that are different with respect to Kusto, like with/without mapping, compress/uncompressed - those are the interesting things and not the csv vs. tsv.

Also, make it easy for users, and treat csv like it's not the default, meaning set a data format for it too. It makes it easier to understand...